operator/controllers/http/scaler.go

```diff
@@ -0,0 +1,113 @@
+package http
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-logr/logr"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	"github.com/kedacore/http-add-on/pkg/k8s"
+	"github.com/kedacore/http-add-on/pkg/util"
+)
+
+const (
+	externalScaler string = "external-scaler"
+	grpc           string = "grpc"
+)
+
+func createOrUpdateExternalScalerResources(
+	ctx context.Context,
+	logger logr.Logger,
+	cl client.Client,
+	httpss metav1.Object,
+	scheme *runtime.Scheme,
+) error {
+	scalerName := fmt.Sprintf("%s-%s", httpss.GetName(), externalScaler)
+	_, adminServiceName := getInterceptorServiceNames(httpss.GetName())
+	httpSpec := util.GetHTTPScalingSetSpecFromObject(httpss)
+	httpKind := util.GetHTTPScalingSetKindFromObject(httpss)
+
+	logger = logger.WithValues(
+		"reconciler.appObjects",
+		"addObjects",
+		"HTTPScalingSet.name",
+		httpss.GetName(),
+	)
+	// defer SaveStatus(context.Background(), logger, cl, httpss)
+	selector := map[string]string{
+		"http.keda.sh/scaling-set":           httpss.GetName(),
+		"http.keda.sh/scaling-set-component": externalScaler,
+		"http.keda.sh/scaling-set-kind":      string(httpKind),
+	}
+
+	scalerService := k8s.NewService(scalerName, httpss.GetNamespace(), grpc, httpSpec.Scaler.GetPort(), selector)
```

Comment:
what would be namespace here if we are using ClusterSS?

Reply:
The namespace where the add-on is deployed

Reply:
ok, so some more appropriate name would be beneficial for understanding, but that's just a nit :)